src/objects/conf/base.py

```diff
@@ -84,8 +84,9 @@
 # Django setup configuration
 #
 SETUP_CONFIGURATION_STEPS = (
+    "django_setup_configuration.contrib.sites.steps.SitesConfigurationStep",
```

Comment:
Since this step was also added, could you also add data for this to `docker/setup_configuration/data.yml`? 

```yaml
sites_config_enable: true
sites_config:
  items:
  - domain: example.com
    name: Example site
```